src/components/icon/index.tsx

```diff
@@ -34,7 +34,7 @@ export class SmoothlyIcon {
 					this.document
 						? this.document
 								.replace(` width="512" height="512"`, "")
-								.replace(/(<title>)[\w\d\s-]*(<\/title>)/, `<title>${this.toolTip || ""}</title>`)
+								.replace(/(><\/)\w+/, `><title>${this.toolTip || ""}</title></svg`)
```

Comment:
This regex amuses that the given SVG is empty. This might not be a safe assumption for any SVG. 
This should be a safer pattern `(><\/)svg>$` then it will only ever target the outermost closing svg tag.

Reply:
Yes, forgot about that! Thank you.

Reply:
The replace function is also very powerful when used together with regex. No need to change to [this](https://www.typescriptlang.org/play?#code/MYewdgzgLgBFIgDYBUCWAHGBeGAiAtgJ4zIKIAmcGAhLgFCiSwQBuA5tjAOQA8rHAJyQBTLLlT42uGAA98iSGIAWUKOgBcAek0B3PQDodAZn0gBbTQCYADLc39pwRAEMIEManCpG0lqmE6AEIgMmLWMOEArACMljAxlrgAfDzozlBKMORiALIALNEAbDB5eQAcABIAnIXORpb61lUw9Y3N4dbRALT1PZYAajV1DU0tI+0R0b31SvXWw21jix1TcfUscwujraMrfWMAXtIAZqiIiGJg4MLS0EIA1qK4wACuAgLCYFAAwkhmt1AHsIuohUGBhMBnOgxEIXmByACgSCwcIAFYgMEwkBwhEwO4gR5dHSocgZMT1XCaFL2dhJLgwBjgaAwNKqYQCMCcTQACn0ACokgBKbk8AA6NLYSQAJILNHRGUwYB8IC9ELAcPx9B90C5gMJuayoOywAAaGAAAyl0R4UFQUEQwmlAG94Eg0OgAL48TS2+2OqWWc2ChUQET6RAgNjc5WqqCCoA) but its a suggestion. 
(click run and you will see the result)